### ISSUE TITLE
Errors when GrappleJumpBackward to grappled buffer without ID

### ISSUE THREAD

**Issue by roginfarrer** [cbochs/grapple.nvim#1]
When I mark a handful of buffers with `GrappleMark` (no arguments), then execute `GrappleJumpBackward`, I get this error:

```
Error executing Lua callback: ...ite/pack/packer/start/grapple.nvim/lua/grapple/marks.lua:202: Invalid buffer id: 36
stack traceback:
	[C]: in function 'nvim_buf_get_name'
	...ite/pack/packer/start/grapple.nvim/lua/grapple/marks.lua:202: in function 'find_key'
	...ite/pack/packer/start/grapple.nvim/lua/grapple/marks.lua:184: in function 'find_mark'
	...site/pack/packer/start/grapple.nvim/lua/grapple/jump.lua:50: in function '_jump_backward'
	...site/pack/packer/start/grapple.nvim/lua/grapple/jump.lua:18: in function 'jump_backward'
	.../pack/packer/start/grapple.nvim/lua/grapple/commands.lua:47: in function <.../pack/packer/start/grapple.nvim/lua/grapple/commands.lua:46>
```

Do I have to give `GrappleMark` a name or ID? Based on the readme, I thought it was optional. I don't entirely understand why I would name a mark in the first place.

I'm still "grappling" with exactly how this plugin works, so I might be doing something wrong.

No special config, just `require('grapple').setup()`

**Comment by cbochs** [comment]
Thanks for submitting an issue! This should be fixed in 2054dd27860c1cfcd137c75968b47da0b666fb43. Let me know if if you have more issues. Buffers in the jumplist (regardless of whether they've been marked or not) are sometimes invalid and should have been ignored.

> I'm still "grappling" with exactly how this plugin works

Haha. I have identified some pitfalls with the current design and plugin goals. Specifically, that marking is useful and sometimes there's a niche purpose for named marks (i.e. kind of like vim marks with `mA`). However, the context they added to jumping forwards and backwards was very interesting. I'm working to expand the plugin more in that direction right now 🙂

**Comment by cbochs** [comment]
Closing this as completed. Feel free to comment here or open another issue if you find more bugs. Thanks!
